NEWS.md

```diff
@@ -228,6 +228,11 @@ small selection of feature refinements.
 
 * ggplot2 now requires R >= 3.3 (#4247).
 
+* The `ticks.linewidth` and `frame.linewidth` parameters of `guide_colourbar()`
```

Comment:
Can you please move this news bullet to the top?

Reply:
I've moved it to the dev version bullets now.

Reply:
For future reference, we generally put news bullets at the very top.